x-pack/plugins/aiops/public/components/page_header/page_header.tsx

```diff
@@ -71,29 +71,29 @@ export const PageHeader: FC = () => {
   return (
     <EuiPageHeader
       pageTitle={<div css={dataViewTitleHeader}>{dataView.getName()}</div>}
+      rightSideGroupProps={{
+        gutterSize: 's',
+        'data-test-subj': 'aiopsTimeRangeSelectorSection',
+      }}
       rightSideItems={[
-        <EuiFlexGroup gutterSize="s" data-test-subj="aiopsTimeRangeSelectorSection">
-          {hasValidTimeField ? (
-            <EuiFlexItem grow={false}>
-              <FullTimeRangeSelector
-                frozenDataPreference={frozenDataPreference}
-                setFrozenDataPreference={setFrozenDataPreference}
-                dataView={dataView}
-                query={undefined}
-                disabled={false}
-                timefilter={timefilter}
-                callback={updateTimeState}
-              />
-            </EuiFlexItem>
-          ) : null}
-          <DatePickerWrapper
-            isAutoRefreshOnly={!hasValidTimeField}
-            showRefresh={!hasValidTimeField}
-            width="full"
-            flexGroup={false}
+        <DatePickerWrapper
+          isAutoRefreshOnly={!hasValidTimeField}
+          showRefresh={!hasValidTimeField}
+          width="full"
+          flexGroup={false}
+        />,
+        hasValidTimeField && (
```

Comment:
If `hasValidTimeField` is false, we get this:

<img width="1213" alt="image" src="https://github.com/user-attachments/assets/6128401b-e5d7-45bf-a460-89c965f5a11b">

Reply:
Should be fixed now in https://github.com/elastic/kibana/pull/198625/commits/c2b835a34fb6702e35fb5f71b8e9cb4f5d59d56d:

<img width="1180" alt="" src="https://github.com/user-attachments/assets/cea3711e-70e1-44fa-a9d9-786ce1f11289">

Thanks for the help with the repro steps!